index.html

```diff
@@ -8,62 +8,194 @@
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            
+            
+          </div>
+        </section>
+        <section class="about">
+
+		  <div class="about-me">
+		    <div class="heading abt-header">About me</div>
+		    <div class="sub-heading abt-sub-header">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
```

Comment:
Kindly change this to a `p` , paragraph tag. This is for correct semantics.

Reply:
done @Nyame-Wolf